index.html

```diff
@@ -15,224 +16,268 @@
-              don't hestiate to contact me.</p>
-          </div>
-          
-           <div>
-            <ul class="icons-container">
-              <li><a><img src="images/github.png" alt="github"/></a></li>
-              <li><a><img src="images/linkedin.png" alt="linkedin"/></a></li>
-              <li><a><img src="images/angel list.png" alt="angel-list"/></a></li>
-              <li><a><img src="images/twitter.png" alt="twitter"/></a></li>
-              <li><a><img src="images/medium.png" alt="medium"/></a></li>
-            </ul>           
-           
-    </section >
-    <section class="works">
-        <div class="recent-works">
-            <h2 class="recent-work-h">My Recent Works</h2>
-        </div> 
-        <hr class="hr">    
-      
-    
-        <div class="head-work">
-            <img class="image-container" src="images/Img Placeholder.png" alt="project-cover">
-             <h2 class="story">Multi-Post Stories</h2>
-             <p class="dailypost">A daily selection of privately personalized<br> reads; no accounts or sign-ups required.<br> has been the industry's standard dummy<br> text ever since the 1500s, when an<br> unknown printer took a standard dummy<br> text.</p>
-            
-            <ul class="techstack-container">
-                <li>css</li>
-                <li>html</li>
-                <li>Bootstrap</li>
-                <li>Ruby</li>                 
-                        
-            </ul>                    
-           
-         </div>
-         <div class="c"> <a class="project-button">See Project</a></div>
-         
-        <section class="works-section">
-        <div class="work">
-            <h2>Profesional Art Printing Data</h2>
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+      <div class="welcome">
+        <h1>
+          Hey There. I"m Bezalell
+        </h1>
+      </div>
+      <div class="job-desc">
+        <h2>I'm a Software Developer</h2>
+      </div>
+      <div class="intro">
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don't hestiate to contact me.</p>
+      </div>
+      <div class="icons-container">
+        <ul>
+          <li><a><img src="assets/github.png" alt="github"/></a></li>
+          <li><a><img src="assets/linkedin.png" alt="linkedin"/></a></li>
+          <li><a><img src="assets/angel list.png" alt="angel-list"/></a></li>
+          <li><a><img src="assets/twitter.png" alt="twitter"/></a></li>
+          <li><a><img src="assets/medium.png" alt="medium"/></a></li>
+        </ul>
+        </div>
+    </section>  
+      <!-- Recent-Works -->
+    <div class="recent-works">
```

Comment:
- [OPTIONAL] It is required to stick to the design as close as possible. Your first project under my recent work does not look exactly as we have on the figma design. Kindly give it a closer look and work you design alongside the figma design. 

### Your Design
![your-first-work](https://user-images.githubusercontent.com/26597432/222906664-cc871237-4a95-4362-ab90-278cd4bbf204.png)


### Figma Design
![temp-first-work](https://user-images.githubusercontent.com/26597432/222906655-6fd158df-ddcc-4bf9-b906-a627af294e68.png)